core/domain/classifier_services_test.py

```diff
@@ -505,3 +505,34 @@ def test_convert_strings_to_float_numbers_in_classifier_data(self):
             classifier_services.convert_strings_to_float_numbers_in_classifier_data( #pylint: disable=line-too-long
                 test_dict))
         self.assertDictEqual(expected_dict, output_dict)
+
+    def test_can_not_convert_strings_to_float_numbers_in_classifier_data(self):
+        with self.assertRaisesRegexp(
+            Exception, 'Expected all classifier data objects to be '
+            'lists, dicts, strings, integers but received'):
+            classifier_services.convert_strings_to_float_numbers_in_classifier_data( #pylint: disable=line-too-long
```

Comment:
I think you can get rid of the pylint pragma by doing something like
(classifier_services
.convert_strings_to_float_numbers_in_classifier_data(...))

Reply:
Done